tg_bot/modules/stickers.py

```diff
@@ -138,17 +145,26 @@ def kang(update: Update, context: CallbackContext):  # sourcery no-metrics
         return
 
     # User sent /kang in reply to a message
-    if msg.reply_to_message:
-        if msg.reply_to_message.sticker:
-            is_animated = msg.reply_to_message.sticker.is_animated
-            file_id = msg.reply_to_message.sticker.file_id
+    if rep := msg.reply_to_message:
+        if rep.sticker:
+            is_animated = rep.sticker.is_animated
+            is_video = rep.sticker.is_video
+            file_id = rep.sticker.file_id
             # also grab the emoji if the user wishes
             if not args:
-                sticker_emoji = msg.reply_to_message.sticker.emoji
-        elif msg.reply_to_message.photo:
-            file_id = msg.reply_to_message.photo[-1].file_id
-        elif msg.reply_to_message.document:
-            file_id = msg.reply_to_message.document.file_id
+                sticker_emoji = rep.sticker.emoji
+        elif rep.photo:

```

Comment:
aaa i wish server had 3.10 python then we could use switch case here

Reply:
Yep thats why i kept this syntax